misc/community-marketing-kit.md

```diff
@@ -0,0 +1,71 @@
+* [Snippets](https://github.com/mozilla-l10n/engagement-l10n/tree/master/en-US/snippets/2017) and [emails](https://github.com/mozilla-l10n/engagement-l10n/tree/master/en-US/emails/2017): Only eight locales are supported on a regular basis. The team is not ready to expand to more locales due to manual process to push them live. You can leverage the content you see fit and publish them on your community social channels. We are in discussion with the Snippets team to support occasional request for locales not regularly supported. Check [Snippets](https://github.com/mozilla-l10n/localizer-documentation/blob/master/webprojects/snippets.md) documentation for details.
+* Other marketing messages: Occasionally we come across interesting messages through Mozilla social channels. The l10n drivers will share them through familiar communication channels such as Facebook, Telegram, and Twitter.
+
+### Key messages of the new Firefox:
+
+* Firefox is fast, modern, and fun
+* Differentiators between Firefox and the Big Machine
+
+  * Big vs. small
+  * Bloated vs. lean
+  * Being watched vs. having more privacy
+  * Ads vs. content
+
+### Targeted audience: attract, inform and convert:
+
+* Switch English user to target language
+* Conscious choosers
+* New and non-users
+* High value users
+* Lapsed users
+
+## Graphic Assets
+
+## Guidelines and downloadable [designlanguage.mozilla.org]
+
+With the new logo and the new look, we now have [a brand new guideline](designlanguage.mozilla.org) at a new home. If you have resources to create your own printable file, make sure to read the guideline on the DOs and DONTs and adhere to the guidelines and suggestions closely.
+
+### Designs
+
+If you do NOT have resources in tools or talents to create your own images, you can do two things:
+
+* Check to see if what you need is already created at [OpenDesign github repository](https://github.com/mozilla/OpenDesign/tree/master/2017). We will add more images and templates over time
+* Click through some of the open and closed issues to see the kind of work the team has covered
+* If you don’t find what you need or you want to have a customized one, here are the steps:
+
+  * File [an issue](https://github.com/mozilla/OpenDesign/issues) to the OpenDesign team
+  * Specify your need: image, logo, with text, with localized text; your purpose
+  * Size and color
+  * Deadline
+  * A few reiterations will take place before the team hands over the finished work to you
+
+## Spreading the words
+
+Here are a few channels to spread the words of the new Firefox launch. The followings are mere suggestions. They may or may not be applicable or appropriate for every market.
+
+* Social media: Instagram, Facebook, Twitter, Blogs
+* Community: organize meetup, attend other tech meetups, be a speaker
+* Organization: team up with other tech NGOs, schools, universities, and libraries to discuss web or tech literacy); bulletin board
+* Mozilla: Mozillagram will share photos from events
```

Comment:
If we want to mention this, we should go into some details on how that works (how to get access, etc.)